build-tools/packages/build-cli/src/library/repoPolicyCheck/fluidBuildTasks.ts

```diff
@@ -225,6 +225,9 @@ function eslintGetScriptDependencies(
 			const configFile = fs.readFileSync(eslintConfig, "utf8");
 			config = JSON5.parse(configFile);
 		} else {
+			// Ideally this would use import() instead of require() but refactoring this function to deal with the async
+			// import is a larger change that can be done at a later time if needed.
```

Comment:
This feels like a PR comment rather than standing code comment. Reword?
Incidentally async resolver support isn't too big. I wanted it for a new policy; so the support is in #21240.